index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="img/favicon.png" />
+    <title>FCC: Tribute Page</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <main id="main">
+      <header>
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </header>
+      <div id="img-div">
+        <figure>
+          <img
+          id="img"
+          src="img/norman.jpg"
+          alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+          />
+        <figcaption id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of
+          his life-long war on hunger.
+        </figcaption>
+      </figure>
+      </div>   
+      <section id="tribute-info">
+        <h3 id="headline">Here's a time line of Dr. Borlaug's life:</h3>
+        <ul>
+          <li>
+            <strong>1924</strong>
```

Comment:
Use classes for styling dates. Imagine that now I tell you make all dates in red color...